specificatie/persoon.yaml

```diff
@@ -39,6 +39,37 @@ components:
       example: 34
       minimum: 0
       maximum: 150
+    OmschrijvingVerdrag:
+      type: string
+      description: |
+        Omschrijving van het verdrag op basis waarvan de gegevens door een buitenlandse organisatie zijn aangeleverd.
```

Comment:
Omschrijving van het verdrag op basis waarvan een zusterorganisatie in het buitenland de gegevens bij de RNI-deelnemer heeft aangeleverd.